lambda/chunkArray.ts

```diff
@@ -0,0 +1,14 @@
+export const chunkArray = ({
+	array,
+	chunkSize,
+}: {
+	array: Array<unknown>
+	chunkSize: number
+}): Array<unknown> => {
```

Comment:
If you type the function like this, you do not need the type assertion when using it:

```suggestion
export const chunkArray = <Element>({
	array,
	chunkSize,
}: {
	array: Array<Element>
	chunkSize: number
}): Array<Array<Element>> => {
```

Reply:
See https://github.com/hello-nrfcloud/backend/blob/554155d481c6582691a562aaee89ad6ee108cc7e/util/batchArray.ts